cypress/integration/spawn/host.ts

```diff
@@ -130,8 +130,57 @@ describe("Navigating to Spawn Host page", () => {
           "contain.text",
           "Also start any hosts this task started (if applicable)"
         );
-        cy.dataCy("load-from-task-sync").should("be.disabled");
       });
+
+      it("If 'Load data for dist' is unchecked, selecting one of it's children will check it.'", () => {
+        cy.visit(
+          `spawn/host?spawnHost=True&distroId=rhel71-power8-large&taskId=${taskId}`
+        );
+        cy.wait(200);
+        cy.dataCy("parent-checkbox").then((chk) => {
+          if (chk[0].checked) {
+            cy.dataCy("parent-checkbox").click({ force: true });
+          }
```

Comment:
Why is there a conditional in the test? Shouldn't there be an assumption that the form should be in the correct state?

Reply:
i wanted to decouple this test from relying on initial form state, since that could change the outcome of this test